Верстка секции products

TatianaHorbenko opened the pull request:
HTML база+ добавлена иконка закрытия модального окна

TatianaHorbenko committed:
Верстка секции products

HTML база+ добавлена иконка закрытия модального окна

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,251 @@
+      <ul class="products__list">
+
+ <!-- Первая картинка списка -->
+          <li class="products__item products__item-icecream">
```
Картинки по ТЗ декоративные, их нужно будет псевдоэлементами делать

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,251 @@
+ <section class="products" id="products">
+    <div class="container prodacts__container">
```
очепятка в слове products

Oliko136 commented on src/images/sprite_icons.svg:
```diff
@@ -15,5 +15,8 @@
 <symbol id="icon-arrow" viewBox="0 0 32 32">
 <path d="M19.414 27.414l10-10c0.781-0.781 0.781-2.047 0-2.828l-10-10c-0.781-0.781-2.047-0.781-2.828 0s-0.781 2.047 0 2.828l6.586 6.586h-19.172c-1.105 0-2 0.895-2 2s0.895 2 2 2h19.172l-6.586 6.586c-0.39 0.39-0.586 0.902-0.586 1.414s0.195 1.024 0.586 1.414c0.781 0.781 2.047 0.781 2.828 0z"></path>
 </symbol>
+<symbol id="close" viewBox="0 0 32 32">
```
что это за иконка?

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,251 @@
+
+              <!-- Стрелка вправо -->
+
+              <button type="button" class="prodacts__button" data-modal-icecream-open>
```
очепятка в слове products

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,251 @@
+
+      <!-- Стрелка вправо -->
+
+      <button type="button" class="prodacts__button" data-modal-icecoffe-open>
```
очепятка в слове products

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,251 @@
+
+              <button type="button" class="prodacts__button" data-modal-icecream-open>
+                <svg class="products__icon" width="20" height=20">
+                  <use href="./src/images/sprite_icons.svg#icon-arrow-right"></use>
```
src тут не нужно, так как index.html тоже лежит в src

TatianaHorbenko committed:
Ветка products

Убрала опечатки, сменила picture на псевдоэлементы, убрала иконку закрытия модального окна, поменяла путь иконки, начала накидывать стили

TatianaHorbenko committed:
Ветка products

Поменяла некоторые названия и размеры иконки

TatianaHorbenko committed:
Ветка продукты

Убрала лишнее в коде

TatianaHorbenko committed:
Окончательный вариант предыдущие правки не нужны

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,161 @@
+          <li class="products__item products__item-icecream">
+              
+           <h3 class="products__card products__card-icecream">
+             ::before
```
псевдоэлементы делаются через ссс, в разметке они не нужны

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,161 @@
+               </li>
+
+<!--Вторая карточка списка  -->
+<li class="products__item products__item-icecoffe">
```
в слове icecoffee очепятка:)

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,161 @@
+                  <use href="./images/sprite_icons.svg#icon-arrow-right"></use>
+                </svg>
+              </button>
+                </div>
```
это же див от контейнера? почему он тут заканчивается?

Oliko136 commented on src/partials/products.html:
```diff
@@ -0,0 +1,161 @@
+          <use href="./images/sprite_icons.svg#icon-arrow-right"></use>
+        </svg>
+      </button>
+      </div>
```
и тут див закрывающий без открывающего

TatianaHorbenko committed:
Убрала див, псевдоэлементы, опечатки

TatianaHorbenko committed:
Закомментировала модальное окно, чтобы не мешало
